systemctl_status.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# List of status codes:
+# https://www.freedesktop.org/software/systemd/man/systemctl.html#is-system-running
+
+STATE=$(systemctl is-system-running)
+echo "status ok"
+echo "metric systemctl_status string $STATE"
+
+: <<EOF
```

Comment:
This is a cool trick that I didn't know about but...can you instead document the example alarm criteria in the header section's comment block, as mentioned here:

https://github.com/racker/rackspace-monitoring-agent-plugins-contrib#alarm-criteria